docs/get-started/using-with-react.md

```diff
@@ -97,6 +112,30 @@ It is possible to use JSX syntax to create deck.gl layers as React children of t
 > Caveat: The JSX layer syntax is limitated in that it only works when the layers are direct children of the `DeckGL` component. deck.gl layers are not true React components and cannot be rendered independently by React, and the JSX support depends on deck.gl intercepting the JSX generated child elements before React tries to render them.
 
 
+## Using JSX with deck.gl views
+
+It is possible to use JSX syntax to create deck.gl views as React children of the `DeckGL` React components, instead of providing them as ES6 class instances to the `views` prop.
+
+The following code renders the same set of layers in two viewports, splitting the canvas into two columns:
+
+```jsx
+  render() {
+    const layers = [
+      new LineLayer({id: 'line-layer', data})
+    ];
+
+    return (
+      <DeckGL initialViewState={initialViewState} layers={layers} >
+        <MapView id="map" width="50%" controller={MapController}>
+          {this._renderMap}
+        </MapView>
```

Comment:
Hmm... Looking at this, I bet that people are going to try to nest layers inside the views...

Reply:
Yeah that could happen...